cli/deno_error.rs

```diff
@@ -50,6 +50,10 @@ pub fn bad_resource() -> ErrBox {
   StaticError(ErrorKind::BadResource, "bad resource id").into()
 }
 
+pub fn bad_resource_kind() -> ErrBox {
+  StaticError(ErrorKind::BadResource, "bad resource kind").into()
```

Comment:
"bad resource kind" is not a very descriptive message - it took me a few minutes to figure out the difference from "bad resource id". 

I question the value of distinguishing between these two types of errors... What do you think about replacing this with the existing `bad_resource()` error?

Reply:
Yeah it makes sense :+1: I'll replace them